server/src/main/java/org/eclipse/lsp/cobol/service/delegates/references/ElementOccurrences.java

```diff
@@ -17,118 +17,107 @@
 import lombok.NonNull;
 import lombok.Value;
 import org.eclipse.lsp.cobol.core.model.tree.Context;
-import org.eclipse.lsp.cobol.core.semantics.outline.RangeUtils;
 import org.eclipse.lsp.cobol.service.CobolDocumentModel;
 import org.eclipse.lsp.cobol.service.delegates.validations.AnalysisResult;
 import org.eclipse.lsp4j.*;
 
 import javax.annotation.Nullable;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
 import java.util.function.Function;
 import java.util.function.Predicate;
-import java.util.function.Supplier;
 import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
 import static org.eclipse.lsp.cobol.service.PredefinedCopybooks.PREF_IMPLICIT;
 import static org.eclipse.lsp.cobol.service.utils.SyntaxTreeUtil.findNodeByPosition;
 
 /**
  * This occurrences provider resolves the requests for the semantic elements based on its positions.
+ * (new Element(Collections.emptyList(), Collections.emptyList()))
  */
 public class ElementOccurrences implements Occurrences {
-    @Override
-    public @NonNull List<Location> findDefinitions(@Nullable CobolDocumentModel document, @NonNull TextDocumentPositionParams position) {
-        if (document == null) return Collections.emptyList();
-        return findElementByPosition(document, position).definitions;
-    }
-
-    @Override
-    public @NonNull List<Location> findReferences(@Nullable CobolDocumentModel document, @NonNull TextDocumentPositionParams position, @NonNull ReferenceContext context) {
-        if (document == null) return Collections.emptyList();
-        Element element = findElementByPosition(document, position);
-        List<Location> references = element.usages;
-        if (context.isIncludeDeclaration()) {
-            references = new ArrayList<>(references);
-            references.addAll(element.definitions);
-        }
-        return references;
-    }
-
-    @Override
-    public @NonNull List<DocumentHighlight> findHighlights(@Nullable CobolDocumentModel document, @NonNull TextDocumentPositionParams position) {
-        if (document == null) return Collections.emptyList();
-        return findReferences(document, position, new ReferenceContext(true)).stream()
-                .filter(byUri(position))
-                .map(toDocumentHighlight())
-                .collect(Collectors.toList());
-    }
-
-    private static Element findElementByPosition(CobolDocumentModel document, TextDocumentPositionParams position) {
-        AnalysisResult result = document.getAnalysisResult();
-        Optional<Element> fromTree = Optional.ofNullable(result.getRootNode())
+  private static Element findElementByPosition(
+      CobolDocumentModel document, TextDocumentPositionParams position) {
+    AnalysisResult result = document.getAnalysisResult();
+    Optional<Element> fromTree =
```

Comment:
This variable is not needed. Please, inline the call.

Reply:
done